src/main/java/bio/terra/service/dataset/DatasetSummary.java

```diff
@@ -64,4 +69,22 @@ public DatasetSummary createdDate(Instant createdDate) {
         this.createdDate = createdDate;
         return this;
     }
+
+    public List<StorageResource> getStorage() {
+        return storage;
+    }
+
+    public DatasetSummary storage(List<StorageResource> storage) {
+        this.storage = storage;
+        return this;
+    }
+
+    public String getStorageResourceRegion(GoogleCloudResource storageResource) {
+        return this.storage.stream()
```

Comment:
I don't think TDR code normally uses `this.`, except in setters and constructors where there's a name collision.
```suggestion
        return storage.stream()
```

Reply:
Every other method in the class uses `this` to reference internal fields.

Reply:
Yes, they are all setters. It is customary to use `this.` in setters, but not in other methods like this, where there is no name collision.